common/djangoapps/student/management/commands/recover_account.py

```diff
@@ -76,21 +76,21 @@ def handle(self, *args, **options):
 
         for row in csv_reader:
             username = row['username']
-            email = row['email']
-            new_email = row['new_email']
+            current_email = row['current_email']
+            desired_email = row['desired_email']
 
             try:
-                user = get_user_model().objects.get(Q(username__iexact=username) | Q(email__iexact=email))
-                user.email = new_email
+                user = get_user_model().objects.get(Q(username__iexact=username) | Q(email__iexact=current_email))
+                user.email = desired_email
                 user.save()
-                self.send_password_reset_email(user, email, site)
-                successful_updates.append(new_email)
+                self.send_password_reset_email(user, desired_email, site)
```

Comment:
I believe now we can simply use the `user.email` in `send_password_reset_email()` and remove this extra argument `desired_email` since it's already updated on user object.